integ-test/src/test/java/org/opensearch/sql/ppl/SystemFunctionIT.java

```diff
@@ -26,49 +26,60 @@ public void init() throws IOException {
 
   @Test
   public void typeof_sql_types() throws IOException {
-    JSONObject response = executeQuery(String.format("source=%s | eval "
-        + "`str` = typeof('pewpew'), `double` = typeof(1.0),"
-        + "`int` = typeof(12345), `long` = typeof(1234567891011), `interval` = typeof(INTERVAL 2 DAY)"
-        + " | fields `str`, `double`, `int`, `long`, `interval`",
-            TEST_INDEX_DATATYPE_NUMERIC));
+    JSONObject response =
+        executeQuery(
+            String.format(
+                "source=%s | eval `str` = typeof('pewpew'), `double` = typeof(1.0),`int` ="
```

Comment:
is there a way to avoid reformatting the strings here?

Reply:
Yes, fixed.